learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -9,12 +9,144 @@
 MeiliSearch has been designed to be a ready-to-go open source solution and deliver an intuitive and instantaneous search experience. As the amount of information available on the websites increases as time goes by, it is essential to provide users with the most relevant results. Since many closed source search engines and open source alternatives can be used, determining which one to install is critical and may not seem easy at first.
 
 The present article covers a comparison of alternatives to MeiliSearch, i.e., available search technologies to be considered when planning to implement a search bar.
+The [comparison table](/learn/what_is_meilisearch/comparison_to_alternatives.md#comparison-table) gives a good overview of how MeiliSearch is different from its alternatives.
+The [approach comparison](/learn/what_is_meilisearch/comparison_to_alternatives.md#approach-comparison) gives details about how MeiliSearch measures against [ElasticSearch](/learn/what_is_meilisearch/comparison_to_alternatives.md#meilisearch-vs-elasticsearch) and [Algolia](/learn/what_is_meilisearch/comparison_to_alternatives.md#meilisearch-vs-algolia).
+This article ends with [a quick look at the search engine landscape](/learn/what_is_meilisearch/comparison_to_alternatives.md#a-quick-look-at-the-search-engine-landscape).
 
 ::: note
 Please be advised that many of the search products described below are constantly evolving—just like MeiliSearch. These are only our own impressions, and may not reflect recent changes. If something appears inaccurate, please don't hesitate to open an [issue or pull request](https://github.com/meilisearch/documentation).
 :::
 
-## Comparisons
+## Comparison table
+
+### General Overview
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:----:|:----:|:-----:|:----:|
+| Source code licensing | [MIT](https://choosealicense.com/licenses/mit/) <br> (Fully open-source) |  Closed-source  | [GPL-3](https://choosealicense.com/licenses/gpl-3.0/) <br> (Fully open-source) | SSPL <br>([Not open-source](https://opensource.org/node/1099))  |
+| Built with | Rust <br> [Check out why we believe in Rust](https://www.abetterinternet.org/docs/memory-safety/). | C++ | C++ | Java |
+| Data storage | Disk with Memory Mapping -- Not limited by RAM | Limited by RAM | Limited by RAM | Disk with RAM cache |
+
+### Features
+
+#### Connect
+
+We list only officially supported libraries by the internal teams of the different projects.
```

Comment:
On second thought, I feel like _teams of each different project_ is misleading, it can be confused with the official Golang, Swift, RoR teams...
How about: 
> we are only listing libraries officially supported by the internal teams of each search engines that are compared here.

Reply:
@guimachiavelli lmk how this sounds, thanks :)